cmd/jobsList.go

```diff
@@ -75,7 +75,7 @@ func init() {
 
 	jobsCmd.AddCommand(lsCmd)
 
-	jobsCmd.PersistentFlags().StringVar(&commandLineInput.withStatus, "with-status", "All",
+	lsCmd.PersistentFlags().StringVar(&commandLineInput.withStatus, "with-status", "All",
```

Comment:
This was showing up as a flag that applied to every sub-command. Instead, the `jobs show` command actually had its definition (which has the same flag for transfers not jobs) overwritten.